src/app.css

```diff
@@ -0,0 +1,179 @@
+  margin: 0;
+  padding: 0;
+  min-width: 320px;
+  background: #fafafa;
+}
+#root {
+  box-sizing: border-box;
+  max-width: 640px;
+  width: 100%;
+  margin: 0 auto;
+  padding: 20px;
+  border-radius: 5px;
+  text-align: center;
+}
+header {
+  text-align: center;
+  margin-bottom: 20px;
+}
+section {
+  text-align: center;
+  margin: 20px 0;
+}
+h1 {
+  font-size: 42px;
+  font-weight: normal;
+  letter-spacing: 5px;
+  margin: 0;
+}
+h2 {
+  font-size: 14px;
+  font-weight: normal;
+  margin: 0;
+}
+h3 {
+  margin: 10px 0;
+}
+a {
+  color: #00bcd4;
+}
+.key_entry {
+  display: inline-block;
+  margin: 0 10px 5px 0;
+  font-size: 14px;
+}
+.search_form {
+  display: inline-block;
+  width: 270px;
+  text-align: left;
+  position: relative;
+}
+.search_label {
+  display: block;
+  font-size: 14px;
+  margin-bottom: 5px;
+}
+.search_field_container {
+  width: 100% !important;
+}
+.search_field {
+  background: #eeeeee;
+  height: 40px !important;
+  transition: background 0.25s ease;
+}
+.search_field_focused {
+  background: #e0f7fa;
+  transition: background 0.25s ease;
+}
+.search_field:before {
+  border-bottom-width: 0 !important;
+}
+.search_field:after {
+  border-bottom-color: #00bcd4 !important;
+}
+.search_input {
+  padding: 10px !important;
+}
+.swap_button {
+  display: inline-block;
+  width: 40px;
+  height: 40px;
+  margin: 0 10px !important;
+  padding: 0 !important;
+  vertical-align: bottom !important;
+}
+.search_results_popper {
+  width: 100% !important;
+  margin-top: 10px !important;
+  z-index: 10 !important;
+}
+.search_results_paper {
+  max-height: calc(4 * 40px);
+  overflow-y: auto;
+  border-radius: 0 !important;
+}
+.search_results_item {
+  height: 40px !important;
+  padding: 0 10px !important;
+  font-size: 14px !important;
+}
+.search_results_item_metanode {
+  margin-right: 5px;
+}
+.metanode_chip {
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+  min-width: 20px;
+  min-height: 20px;
+  width: 20px;
+  height: 20px;
+  border-radius: 10px;
+  font-weight: 900;
+  font-size: 14px;
+  letter-spacing: 1px;
+  background: #616161;
+  color: #ffffff;
+}
+.metanode_chip[data-metanode="Gene"] {
+  background: #2196f3;
+}
+.metanode_chip[data-metanode="Compound"] {
+  background: #ff9800;
+}
+.metanode_chip[data-metanode="Anatomy"] {
+  background: #e91e63;
+}
+.metanode_chip[data-metanode="Disease"] {
+  background: #4caf50;
+}
+.metanode_chip[data-metanode="Side Effect"] {
+  background: #3f51b5;
+}
+.metanode_chip[data-metanode="Symptom"] {
+  background: #9c27b0;
+}
+.metanode_chip[data-metanode="Biological Process"] {
+  background: #009688;
+}
+.metanode_chip[data-metanode="Cellular Component"] {
+  background: #795548;
+}
+.metanode_chip[data-metanode="Molecular Function"] {
+  background: #00bcd4;
+}
+.metanode_chip[data-metanode="Pathway"] {
+  background: #607d8b;
+}
+.metanode_chip[data-metanode="Pharmalogical Class"] {
+  background: #ff5722;
+}
```

Comment:
Can you use the node [fill colors here](https://github.com/hetio/hetionet/blob/b7c144da0dd428def9e3c262d0bc48c050cf631d/hetnet/neo4j/docker/guides/graphstyle.grass#L20-L94)? These are currently used for the Neo4j browser and existing visualizations.

Reply:
I'd recommend not using those colors if at all possible, because they're a little inconsistent and have duplicates among the node types. Wouldn't having distinct colors for each thing be more user friendly? 

I could certainly match my equivalent colors where possible, ie compound = red, disease = brownish, anatomy = green

Reply:
The duplicates are intentional because they all come from the Gene Ontology. It's important that we keep using the same color scheme.